pkg/controller/disruption/disruption.go

```diff
@@ -588,7 +589,7 @@ func (dc *DisruptionController) getExpectedScale(pdb *policy.PodDisruptionBudget
 			var controllerNScale *controllerAndScale
 			controllerNScale, err = finder(pod)
 			if err != nil {
-				return
+				continue
```

Comment:
With this change, any errors from the finders would fail silently. And if the correct finder returns an error, the controller would incorrectly log and create an event that the pod does not have a controller. I think it is better that this function returns an error and then the reconciliation loop will run again and then it will hopefully succeed.
Also, it seems like none of the finders ever return anything else than nil for the error.

Reply:
```
that this function returns an error and then 
```
The err assigned on line 591 would be returned